libtiledbsoma/src/soma/soma_dense_ndarray.cc

```diff
@@ -98,7 +98,7 @@ SOMADenseNDArray::SOMADenseNDArray(
     array_ = std::make_shared<SOMAArray>(
         mode,
         uri,
-        "unnamed",  // name
+        uri,  // label used when debugging
```

Comment:
This value displayed when we do spdlog which may unduly enlarge the logging line. I am wondering if 'somewhere' (here, or on the loggiing side?) we should maybe break this into tokens (ie from, say, s3://some/user/bucket to just bucket) to display something more concise ?  Else we can of course change the default logging format and drop this 'name' field..  Or make it a parameter.  Or keep the old constant.

Reply:
Makes sense - I will tokenize the filepath and grab just the filename then.

Reply:
That would be a nice net improvement. Screenshot to make the color stand out -- we would then see the key part of URI:

![image](https://github.com/single-cell-data/TileDB-SOMA/assets/673121/8d414306-2b14-478b-b59e-0bbff6d8586e)